lib/wsdl.js

```diff
@@ -523,7 +529,9 @@ MessageElement.prototype.postProcess = function(definitions) {
 
       if (typeNs) {
         if (type.name in Primitives) {
+          if (Primitives.hasOwnProperty(type.name)) {
             // this.element = this.element.$type;
```

Comment:
I think we just found some "dead" code here so I don't think we need a guard here but rather look what caused this code to be commented out and left behind 🌵